schemas/aws/terraform-resource-2.yml

```diff
@@ -454,6 +465,28 @@ oneOf:
       "$ref": "/common-1.json#/definitions/vaultSecret"
     annotations:
       "$ref": "/common-1.json#/definitions/annotations"
+    event_notifications:
+      type: array
+      items:
+        type: object
+        properties:
+          destination:
+            type: string
+          source_type:
+            type: string
+            enum:
+            - db-instance
+            - db-security-group
+            - db-parameter-group
+            - db-snapshot
+            - db-cluster
+            - db-cluster-snapshot
+          event_categories:
```

Comment:
So one of each source_type will have its own event_categories. Are you suggesting get all the available categories and have them as enum? I was thinking this can be handled by the terraform layer

Reply:
That's true... I was mostly thinking about the db-instance use case because that's our primary use case at the moment. It probably would be a bit more complicated than necessary to enumerate all of them, so this is probably fine for now.